tools/topology/topology2/cavs/platform/intel/sdw-amp-generic.conf

```diff
@@ -0,0 +1,84 @@
+
+Define {
+	SPK_STREAM 'SDW1-Playback'
+	ALH_2ND_SPK_ID 7
+	SDW_AMP_BE_ID 2
+}
+
+Object.Dai {
+	ALH."514" {
+		id 		$SDW_AMP_BE_ID
+		direction	"playback"
+		name		$SPK_STREAM
+		default_hw_conf_id	0
+		rate			48000
+		channels		2
+
+		Object.Base.hw_config."ALH514" {
+			id	0
+		}
+	}
+}
+
+Object.Pipeline {
+	host-copier-gain-mixin-playback."3" {
+		index 3
+
+	Object.Widget.copier.1 {
+		stream_name "sdw amplifiers"
+	}
+	Object.Widget.gain.1 {
+		Object.Control.mixer.1 {
+			name 'Amplifier Volume'
+			}
+		}
+	}
```

Comment:
too many closing braces, does this work?

Reply:
> too many closing braces, does this work?

Thanks @plbossart This is an indentation issue. Fixing it now.